pkg/webhook/operatingsystemconfig/resources/auditrules/11-privileged-special.rules

```diff
@@ -5,4 +5,5 @@
 ## This file is managed by the shoot-rsyslog-relp extension
 ## The original file was moved to /etc/audit/rules.d.original
 
--a exit,always -F arch=b64 -S mount -S mount_setattr -S umount2 -S mknod -S mknodat -S chroot -F auid!=-1 -F key=privileged_special
+-a exit,always -F arch=b64 -S mount -S umount2 -S mknod -S mknodat -S chroot -F auid!=-1 -F key=privileged_special
+-a exit,always -F arch=b64 -S mount_setattr -F auid!=-1 -F key=privileged_special
```

Comment:
Does it make sense to add a comment that mount_setattr is availble only in kernel versions >= 5.12?

Reply:
Yep, makes perfect sense. I've added the comment now